src/Development/Shake/Command.hs

```diff
@@ -145,7 +145,8 @@ commandExplicit funcName oopts results exe args = do
         shelled = runShell (unwords $ exe : args)
 
         ignore = map (?==) shakeLintIgnore
-        ham cwd xs = [makeRelative cwd x | x <- map toStandard xs
+        ham cwd xs = liftIO $ sequence [fromMaybe x <$> makeRelativeEx cwd x
```

Comment:
I'm concerned with performance here. Thoughts?

Reply:
We only call this after spawning a process. Hard to do much that has a performance impact relative to that.